src/main/resources/model/instance.json

```diff
@@ -283,10 +283,11 @@
           "index": "keyword_lowercase"
         },
         "accessionNumber": {
-          "index": "source"
+          "index": "keyword_lowercase"
         },
         "formerIds": {
-          "index": "source"
+          "searchTypes": [ "facet", "filter" ],
```

Comment:
@psmagin , could you please check if index for array element is configured correctly?

Reply:
You can skip `searchTypes` fields.